main.tex

```diff
@@ -108,40 +114,40 @@ \section*{Introduction}
 %
-However, these studies analysed only a small region of the VGSC gene, and therefore had limited power to make inferences about the selective events or spread of resistance alleles. 
+However, these studies analysed only a small gene region in a limited number of mosquito populations, and therefore had limited resolution to make inferences about genetic relationships between gene sequences (haplotypes) carrying this allele. 
 %
 It has also been shown that the \texttt{L995F} allele spread from \textit{An. gambiae} to \textit{An. coluzzii} in West Africa \cite{Clarkson2014,Norris2015}. 
 %
-However, both \texttt{L995F} and \texttt{L995S} now have wide geographical distributions \cite{Silva2014}, and no attempts have been made to reconstruct the geographical spread of either allele.
+However, both \texttt{L995F} and \texttt{L995S} now have wide geographical distributions \cite{Silva2014}, and no attempts have been made to infer or track the geographical spread of either allele.
 %%
 
 %%
 Here we report an in-depth analysis of the VGSC gene, using whole-genome Illumina sequence data from phase 1 of the \emph{Anopheles gambiae} 1000 Genomes Project (Ag1000G) \cite{Ag1000gConsortium2017}.
 %
-We investigate variation across the complete gene coding sequence, to fully characterise the variants potentially driving primary and secondary selective sweeps of target-site resistance to pyrethroids in natural mosquito populations.
+The Ag1000G phase 1 resource includes data on nucleotide variation in samples from 8 countries, with representation of West, Central and East Africa, and of both \textit{An. gambiae} and \textit{An. coluzzii}.
 %
-We then use haplotype data from the chromosomal region spanning the VGSC gene to study the genetic backgrounds carrying resistance alleles.
+We investigate variation across the complete gene coding sequence, and report population genetic data for both known and novel non-synonymous nucleotide substitutions.
 %
-The goal of these analyses is to diagnose how many separate selective events have occurred, which are localised, and which are spreading.
+We then use haplotype data from the chromosomal region spanning the VGSC gene to study the genetic backgrounds carrying resistance alleles, and infer the geographical spread of resistance between mosquito populations.
 %
-Finally we explore ways in which variation data from Ag1000G could be used to design high-throughput, low-cost genetic assays for monitoring pyrethroid resistance, with the capability to differentiate and track separate resistance driven selective sweeps.  
+Finally, we explore ways in which variation data from Ag1000G could be used to design high-throughput, low-cost genetic assays for monitoring pyrethroid resistance, with the capability to differentiate and track separate resistance outbreaks.  
 %%
```

Comment:
I've made a few edits to the intro, mainly attempting to increase readability a little, and also to dial back on our claims to infer the history of spread of resistance, which we have punted to a future paper.

Reply:
Hi @alimanfoo,

I had tried to remove all 'outbreak' instances as this is a novel concept and I feared it might distract from the descriptive biology. Do you think we can put it back in without the explanation that we had before where we suggested to the reader that looking at resistance in an epidemiological light could be beneficial?

Reply:
Happy to remove "outbreak", I was just trying to find something other than "selective sweep" which is a very ambiguous term. This was the only place where I was going to use "outbreak" in an intuitive sense. Any suggestions?

Reply:
Hmmm, I couldn't come up with anything useful when I was re-writing, but agree that selective sweep isn't perfect (and is super clunky). Insecticide resistance outbreaks/events/inception/origin...

Perhaps folk will have less difficulty if 'outbreak' is bring used intuitively, rather than unequivocally like we did in the previous draft with the epidemiology angle explanation?